src/components/notes/hooks/useReferencedFolderForNote.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react'
+import { CozyFolder } from 'cozy-doctypes'
+import { getFullLink } from 'lib/utils'
+
+const notesFolderDocument = {
+  _type: 'io.cozy.apps',
+  _id: 'io.cozy.apps/notes'
+}
+
+const useReferencedFolderForNote = client => {
+  const [notesFolder, setNotesFolder] = useState(null)
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const driveUrl = getFullLink(client)
+      try {
+        setNotesFolder(driveUrl)
+        const referencedFolder = await CozyFolder.getReferencedFolders(
+          notesFolderDocument
+        )
+        const folderUrl = getFullLink(client, referencedFolder[0]._id)
```

Comment:
I know `getFullLink ` was here before, but seeing it in this context I realize that the name is not very clear. It's really not obvious that it generates a link to the drive app :/. I think renaming it would be a good idea.